pkg/storegateway/series_refs_test.go

```diff
@@ -1749,7 +1751,7 @@ func BenchmarkOpenBlockSeriesChunkRefsSetsIterator(b *testing.B) {
 				b.Cleanup(cancel)
 
 				var block = newTestBlock()
-				indexReader := block.indexReader()
+				indexReader := block.indexReader(worstCaseFetchedDataStrategy{1.0})
```

Comment:
Should we rather use `selectAllStrategy{}` here, given we want to stress `openBlockSeriesChunkRefsSetsIterator()`?

Reply:
`openBlockSeriesChunkRefsSetsIterator` does postings as well. I changed it here so that we can see the tradeoff in the increased number of series fetched.

I can revert it now to `selectAll` and then we can turn it back to `worseCase` when/if that strategy becomes the default.